tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -2731,6 +2731,37 @@ port::Status CudnnSupport::DoPrepareForConvolution(
   return port::Status::OK();
 }
 
+port::Status CudnnSupport::IntegerConvolutionSanityChecks(
```

Comment:
In the spirit of making cudan_dnn.cc as thin of a wrapper as possible, I think this sanity check should be in XLA, e.g. RunCudnnConvolutionImpl.

Reply:
I will try to move the sanity checks.

Reply:
No need for this change.